lib/containers/bats.pm

```diff
@@ -248,12 +248,24 @@ sub bats_post_hook {
     script_run('dmesg > dmesg.txt');
     script_run('findmnt > findmnt.txt');
     script_run('rpm -qa | sort > rpm-qa.txt');
+    script_run('sysctl -a > sysctl.txt');
     script_run('systemctl > systemctl.txt');
     script_run('systemctl status > systemctl-status.txt');
     script_run('systemctl list-unit-files > systemctl_units.txt');
     script_run('journalctl -b > journalctl-b.txt', timeout => 120);
     script_run('tar zcf containers-conf.tgz $(find /etc/containers /usr/share/containers -type f)');
 
+    for my $ip_version (4, 6) {
+        script_run("ip -$ip_version addr > ip$ip_version-addr.txt");
+        script_run("ip -$ip_version route > ip$ip_version-route.txt");
+    }
+    script_run("iptables-save > iptables.txt");
+    script_run("ip6tables-save > ip6tables.txt");
+    script_run('nft list ruleset > nft.txt');
+
+    # Remove all empty logs
+    script_run "find $log_dir -type f -size 0 -exec rm -f {} +";
```

Comment:
Not sure, but this part appears unnecessary to me. We're not concerned about storage space in general, and uploading an empty log is essentially the same as not uploading anything.

Reply:
This is not about storage space but not wasting time clicking on empty links.

Reply:
I don't see the point - one does typically go for the logs that one needs. Here it doesn't matter much, if the logs are not there or are empty.

But in the end it doesn't make a difference, so let's just leave it.